tests/check/check_h26xsigned_auth.c

```diff
@@ -1984,25 +1978,23 @@ START_TEST(public_key_in_sei_and_on_validation_side_later)
   sv_rc = signed_video_add_nalu_and_authenticate(sv_vms, sei->data, sei->data_size, &auth_report);
   ck_assert_int_eq(sv_rc, SV_OK);
   // Set public key
-  sv_rc = signed_video_set_public_key(sv_vms, algo, sv_camera->signature_info->public_key, sv_camera->signature_info->public_key_size);
+  sv_rc = signed_video_set_public_key(sv_vms, sv_camera->signature_info->public_key, sv_camera->signature_info->public_key_size);
```

Comment:
Move the use of `i_nalu` to after this line. That way we secure that it is still possible to continue the operations.